libgnucash/scm/price-quotes.scm

```diff
@@ -529,5 +529,5 @@ Run 'gnc-fq-update' as root to install them.")))
       (gnc:msg "Found Finance::Quote version " (car sources))
       (gnc-quote-source-set-fq-installed (car sources) (cdr sources)))
      (else
-      (display "No Finance::Quote found\n")
-      (gnc:msg "No Finance::Quote found")))))
```

Comment:
This seems to be somewhat of an odd mixup of functions and localizations to me:
If I read the code correctly all of these messages will be going to stdout, which means they well end up in the trace file. Do we want trace file messages to be translated ?

Secondly, the format/display commands and the gnc:msg commands will both output a very similar message to the trace file. The differences being: the former does it unconditionally and with less decoration, the latter will only do so if gnc.scm is set to info on the command line or in log.conf. In addition gnc:msg will add a timestamp and a source file hint.

This has probably grown that way historically: originally there probably was only gnc:msg and then it turned out we needed this information regularly from our users so an additional line was added to add this info unconditionally (and translated apparently) to the trace file.

Somewhere in the 2.6 series I have changed the About dialog to also display the F::Q version that is detected. I believe that is where our users would prefer to look for this info.
The trace file on the other hand is targeted at the devs or other people offering support to users. We can debate whether it's useful or not to keep printing the info in there. I think it would still be useful to keep the F::Q version in there as it brings useful detail together in one file (and it might be useful to extend this with additional info like the gnucash build id). But I don't think it should be translated and honestly I don't care if it's a full sentence in the logs. It's not user oriented.